server/ws/websockets.go

```diff
@@ -270,10 +291,10 @@ func (ws *Server) addListener(wsSession *websocketSession, command *WebsocketCom
 func (ws *Server) removeListener(client *websocket.Conn) {
```

Comment:
nit: probably we can make this method receiving also a `wsClient` instead of `websocket.Conn`, that simplifies the comparison later, and unifies all the code.

Reply:
Done.